stl/inc/format

```diff
@@ -1852,6 +1852,8 @@ struct _Back_insert_iterator_container_access : back_insert_iterator<_Container>
     using back_insert_iterator<_Container>::container;
 };
 
+#pragma warning(push)
+#pragma warning(disable : 4265) // non-virtual destructor in base class

```

Comment:
STL thinks this bug has been fixed already. It may not be in a public preview yet, no action requested from you.

Reply:
STL looked and this shipped in 17.1p3, so this can be removed.